iset.mm

```diff
@@ -99854,6 +99854,84 @@ the previous value and the value of the input sequence (second operand).
       NYNUYLUYKUUIXRXAVWFWNWOWPVURWPXTYAYDYEYFYGYHBCFIJKYIYJ $.
   $}
 
+  ${
+    $d .+ a b k x y $.  $d .+ c k x y $.  $d .+ c n x y $.  $d .+ a b u x y $.
+    $d .+ k w x y z $.  $d F a b k x y $.  $d F c k x y $.  $d F c n x y $.
+    $d F a b u x y $.  $d F k w x y z $.  $d M a b k x y $.  $d M c k x y $.
+    $d M c n x y $.  $d M a b u x y $.  $d M k w x y z $.  $d R a b k x y $.
+    $d R c k x y $.  $d R c n x y $.  $d R k w x y z $.  $d S a b u x y $.
+    $d S u w x y z $.  $d T x y $.  $d a b k ph x y $.  $d c k ph x y $.
+    $d n ph x y $.  $d ph u w x y z $.
+    seq3val.m $e |- ( ph -> M e. ZZ ) $.
+    seq3val.r $e |- R = frec (
+        ( x e. ( ZZ>= ` M ) , y e. _V |-> <. ( x + 1 ) ,
+          ( x ( z e. ( ZZ>= ` M ) , w e. S |->
+            ( w .+ ( F ` ( z + 1 ) ) ) ) y ) >. ) ,
+          <. M , ( F ` M ) >. ) $.
+    seq3val.f $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. S ) $.
+    seq3val.pl $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x .+ y ) e. S ) $.
+    $( Value of the sequence builder function.
+
+       This helps us expand the definition although there should be little need
+       for it once we have proved ~ seqf , ~ seq3-1 and ~ seq3p1 , as further
+       development can be done in terms of those.
+
+       (Contributed by Mario Carneiro, 24-Jun-2013.)  (Revised by Jim Kingdon,
```

Comment:
Probably no line breaks needed.  I say this considering the pages mmtheoremsXXX.html, where they tend to add up.  Of course, that's a judgment call.

Reply:
Probably "us" is not needed either.

Reply:
Changed.